tools/ci/aws_ssm_get_wrapper.sh

```diff
@@ -19,4 +19,4 @@ while [[ $retry_count -lt $max_retries ]]; do
 done
 
 echo "Failed to retrieve parameter after $max_retries retries"
-exit 1
+exit 666
```

Comment:
Did you test that it was working as expected?
If I remember correctly some time ago GitlabCI script was always exiting with 1 even if your script was explicitly exiting with another exit code. That might have been fixed. But I am not 100% sure

Reply:
I did not test this. Let me give it a shot

Reply:
You're right does not seem to work :(

Reply:
Turns out this feature was added in 16.11, which we are not running yet.